### PR TITLE
Add makie latex string note

### PR DESCRIPTION
**Pull request by jeremiahpslewis**
~Based on [this question](https://julialang.slack.com/archives/C6A044SQH/p1663684625489519), trying to make the docs a bit clearer on LatexString handling.~

Updated to 'stress test' the new Latex support in `MathTeXEngine.jl v0.5+`.

### CODE REVIEW

**Review by jkrumbiegel — APPROVED** [review]

File: docs/documentation/latex.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
...
 Makie can render LaTeX strings from the [LaTeXStrings.jl](https://github.com/stevengj/LaTeXStrings.jl) package using [MathTeXEngine.jl](https://github.com/Kolaru/MathTeXEngine.jl/).
 
-This engine supports a subset of LaTeX's most used commands, which are rendered quickly enough for responsive use in GLMakie.
+While this engine is responsive enough for use in GLMakie, it only supports a subset of LaTeX's most used commands.
+
+!!! note
+    Makie is able to render the subset of Latex strings supported by [MathTeXEngine.jl](https://github.com/Kolaru/MathTeXEngine.jl/).
+    Alternative scripts like ``\mathcal{X}`` or ``\mathbb{R}`` are not supported as of `MathTeXEngine.jl` `v0.5.1`.
+    In many cases, these characters can be replaced by unicode equivalents, e.g. ``ℝ`` for ``\mathcal{R}``.
+    A full list of Unicode replacements can be found [here](http://milde.users.sourceforge.net/LUCR/Math/unimathsymbols.pdf).
```
Comment (jkrumbiegel) [inline docs/documentation/latex.md:11]: only thing, I would not link with the word `here`, it's more user friendly to write out at least part of the url or a description

**Comment by jkrumbiegel** [comment]
Have a look at the other link in the section to see how markdown links are written

**Comment by greimel** [comment]
Did you try this on Makie#master? 

https://github.com/Kolaru/MathTeXEngine.jl/blob/50f8b4cbe2b5b477f1ff59b70f119ae97c49041f/prototype/prototype.jl#L162

It seems that `mathbb` etc should work on MathTeXEngine@0.5, which isn't compatible with the latest release of Makie.

cc @Kolaru

**Comment by jeremiahpslewis** [comment]
@greimel Do you have a full reproducible example? I'm on 

```Julia
Status `/private/var/folders/7c/hm96b6vx3r9dc4_s5f6y9m1r0000gn/T/jl_J8s9XE/Project.toml`
  [13f3f980] CairoMakie v0.8.13 `https://github.com/JuliaPlots/Makie.jl.git:CairoMakie#master`
  [5ae59095] Colors v0.12.8
  [ee78f7c6] Makie v0.17.13 `https://github.com/JuliaPlots/Makie.jl.git#master`
  [0a4f8689] MathTeXEngine v0.5.1
```

and get errors when trying to render Latex `\mathbb`.

**Comment by jeremiahpslewis** [comment]
@greimel Do you have a full reproducible example? I'm on 

```Julia
Status `/private/var/folders/7c/hm96b6vx3r9dc4_s5f6y9m1r0000gn/T/jl_J8s9XE/Project.toml`
  [13f3f980] CairoMakie v0.8.13 `https://github.com/JuliaPlots/Makie.jl.git:CairoMakie#master`
  [5ae59095] Colors v0.12.8
  [ee78f7c6] Makie v0.17.13 `https://github.com/JuliaPlots/Makie.jl.git#master`
  [0a4f8689] MathTeXEngine v0.5.1
```

and get errors when trying to render Latex `\mathbb`.

**Comment by greimel** [comment]
Does plotting without latex work for you? You generally have to checkout `MakieCore#master` as well.

This works for me
```julia
begin
	using Pkg
	Pkg.activate(temp = true)
	pkg"add MakieCore#master Makie#master CairoMakie#master"
	using CairoMakie
end

scatter(rand(10), rand(10), axis = (; title = L"this is $\mathbb{E}$"))
```

**Comment by jeremiahpslewis** [comment]
Works for me in Pluto.jl, but fails in the command line with the following error:

```Julia
julia> scatter(rand(10), rand(10), axis = (; title = L"this is $\mathbb{E}$"))
ERROR: ArgumentError: font modifier bb not supported for the current font family.
Stacktrace:
  [1] get_font(state::MathTeXEngine.LayoutState, char_type::Symbol)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout_context.jl:25
  [2] TeXChar(char::Char, state::MathTeXEngine.LayoutState, char_type::Symbol)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/texelements.jl:146
  [3] tex_layout(expr::TeXExpr, state::MathTeXEngine.LayoutState)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:29
  [4] tex_layout(expr::TeXExpr, state::MathTeXEngine.LayoutState)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:97
  [5] _broadcast_getindex_evalf
    @ ./broadcast.jl:683 [inlined]
  [6] _broadcast_getindex
    @ ./broadcast.jl:656 [inlined]
  [7] getindex
    @ ./broadcast.jl:610 [inlined]
  [8] copy
    @ ./broadcast.jl:912 [inlined]
  [9] materialize(bc::Base.Broadcast.Broadcasted{Base.Broadcast.DefaultArrayStyle{1}, Nothing, typeof(MathTeXEngine.tex_layout), Tuple{Vector{TeXExpr}, Vector{MathTeXEngine.LayoutState}}})
    @ Base.Broadcast ./broadcast.jl:873
 [10] tex_layout(expr::TeXExpr, state::MathTeXEngine.LayoutState)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:128
 [11] tex_layout(expr::TeXExpr, font_family::MathTeXEngine.FontFamily)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:18
 [12] _broadcast_getindex_evalf
    @ ./broadcast.jl:683 [inlined]
 [13] _broadcast_getindex
    @ ./broadcast.jl:656 [inlined]
 [14] getindex
    @ ./broadcast.jl:610 [inlined]
 [15] copy
    @ ./broadcast.jl:912 [inlined]
 [16] materialize
    @ ./broadcast.jl:873 [inlined]
 [17] generate_tex_elements(str::LaTeXString, font_family::MathTeXEngine.FontFamily)
    @ MathTeXEngine ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:304
 [18] generate_tex_elements
    @ ~/.julia/packages/MathTeXEngine/XFmQJ/src/engine/layout.jl:298 [inlined]
 [19] texelems_and_glyph_collection(str::LaTeXString, fontscale_px::Float32, halign::Symbol, valign::Symbol, rotation::Quaternion{Float64}, color::RGBA{Float32}, strokecolor::RGBA{Float32}, strokewidth::Int64, word_wrap_width::Int64)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/basic_recipes/text.jl:156
 [20] _get_glyphcollection_and_linesegments(latexstring::LaTeXString, index::Int64, ts::Float32, f::FreeTypeAbstraction.FTFont, al::Tuple{Symbol, Symbol}, rot::Quaternion{Float64}, jus::MakieCore.Automatic, lh::Float64, col::RGBA{Float32}, scol::RGBA{Float32}, swi::Int64, www::Int64)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/basic_recipes/text.jl:73
 [21] call_composed
    @ ./operators.jl:1020 [inlined]
 [22] call_composed
    @ ./operators.jl:1019 [inlined]
 [23] #_#97
    @ ./operators.jl:1016 [inlined]
 [24] ComposedFunction
    @ ./operators.jl:1016 [inlined]
 [25] macro expansion
    @ ~/.julia/packages/Makie/Ppzqh/src/utilities/utilities.jl:201 [inlined]
 [26] broadcast_foreach(::ComposedFunction{Makie.var"#push_args#1815"{Vector{Int64}, Vector{RGBA{Float32}}, Vector{Float32}, Vector{Point{2, Float32}}, Vector{Makie.GlyphCollection}}, typeof(Makie._get_glyphcollection_and_linesegments)}, ::LaTeXString, ::UnitRange{Int64}, ::Float32, ::FreeTypeAbstraction.FTFont, ::Tuple{Symbol, Symbol}, ::Quaternion{Float64}, ::MakieCore.Automatic, ::Float64, ::RGBA{Float32}, ::RGBA{Float32}, ::Int64, ::Int64)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/utilities/utilities.jl:187
 [27] (::Makie.var"#1811#1814"{Base.RefValue{Vector{Int64}}, Observable{Vector{RGBA{Float32}}}, Observable{Vector{Float32}}, Observable{Vector{Point{2, Float32}}}, Observable{Vector{Makie.GlyphCollection}}})(str::LaTeXString, ts::Float64, f::FreeTypeAbstraction.FTFont, al::Tuple{Symbol, Symbol}, rot::Float64, jus::MakieCore.Automatic, lh::Float64, col::RGBA{Float32}, scol::Tuple{Symbol, Float64}, swi::Int64, www::Int64)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/basic_recipes/text.jl:34
 [28] (::Observables.var"#callback#13"{Makie.var"#1811#1814"{Base.RefValue{Vector{Int64}}, Observable{Vector{RGBA{Float32}}}, Observable{Vector{Float32}}, Observable{Vector{Point{2, Float32}}}, Observable{Vector{Makie.GlyphCollection}}}, NTuple{11, Observable{Any}}})(x::Any)
    @ Observables ~/.julia/packages/Observables/ynr7h/src/Observables.jl:339
 [29] #invokelatest#2
    @ ./essentials.jl:807 [inlined]
 [30] invokelatest
    @ ./essentials.jl:804 [inlined]
 [31] notify(observable::Observables.AbstractObservable)
    @ Observables ~/.julia/packages/Observables/ynr7h/src/Observables.jl:143
 [32] plot!(plot::MakieCore.Text{Tuple{Vector{Point{2, Float32}}}})
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/basic_recipes/text.jl:53
 [33] plot!(scene::Scene, P::Type{MakieCore.Text{Tuple{Point{2, Float32}}}}, attributes::Attributes, input::Tuple{Observable{Point{2, Float32}}}, args::Observable{Tuple{Vector{Point{2, Float32}}}})
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/interfaces.jl:417
 [34] plot!(scene::Scene, P::Type{MakieCore.Text}, attributes::Attributes, args::Observable{Point{2, Float32}}; kw_attributes::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/interfaces.jl:335
 [35] plot!
    @ ~/.julia/packages/Makie/Ppzqh/src/interfaces.jl:302 [inlined]
 [36] #plot!#138
    @ ~/.julia/packages/Makie/Ppzqh/src/interfaces.jl:286 [inlined]
 [37] #text!#52
    @ ~/.julia/packages/MakieCore/rLlRw/src/recipes.jl:38 [inlined]
 [38] initialize_block!(ax::Axis; palette::Nothing)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/makielayout/blocks/axis.jl:458
 [39] initialize_block!(ax::Axis)
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/makielayout/blocks/axis.jl:168
 [40] _block(::Type{Axis}, ::Figure; bbox::Nothing, kwargs::Base.Pairs{Symbol, LaTeXString, Tuple{Symbol}, NamedTuple{(:title,), Tuple{LaTeXString}}})
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/makielayout/blocks.jl:403
 [41] #_#1006
    @ ~/.julia/packages/Makie/Ppzqh/src/makielayout/blocks.jl:279 [inlined]
 [42] plot(::Type{Scatter}, ::Vector{Float64}, ::Vararg{Vector{Float64}}; axis::NamedTuple{(:title,), Tuple{LaTeXString}}, figure::NamedTuple{(), Tuple{}}, kw_attributes::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/Ppzqh/src/figureplotting.jl:50
 [43] scatter(::Vector{Float64}, ::Vararg{Vector{Float64}}; attributes::Base.Pairs{Symbol, NamedTuple{(:title,), Tuple{LaTeXString}}, Tuple{Symbol}, NamedTuple{(:axis,), Tuple{NamedTuple{(:title,), Tuple{LaTeXString}}}}})
    @ MakieCore ~/.julia/packages/MakieCore/rLlRw/src/recipes.jl:34
 [44] top-level scope
    @ REPL[99]:1
```

**Comment by jeremiahpslewis** [comment]
> Is that in a fresh julia session?

Nope. Works now, thanks!

**Comment by jeremiahpslewis** [comment]
@jkrumbiegel This PR took a bit of an interesting turn, looks like the necessary Latex is available, but not yet released as a tagged version. I've changed the docs to include a `\mathbb` and `\mathcal` example.

**Comment by Kolaru** [comment]
`\mathbb`, `\mathcal`, `\mathscr` and `\mathfrak` should work in MathTeXEngine v0.5.1. And it *should* be compatible with Makie on master at least.

**Pull request merged**